### ISSUE TITLE
Bidirectional RNN: forward and backward passing to the middle of the sentence

### ISSUE THREAD

**Issue by bluemonk482** [tensorflow/tensorflow#4733]
Hi all, I have a task that requires me to split a sentence into two and model the two parts respectively using bidirectional rnn, e.g.:

**"I think of A, but I am going to B tomorrow"** split into **"I think of A"** and **"but I am going to B tomorrow"**; forward passing goes from "I" to "A" while backward passing goes from "tomorrow" to "but". This way two outputs are trained using directional rnn and then concatenated before feeding to the softmax layer.

My question is, is there an easier to achieve this using bidirectional_rnn in Tensorflow that I am not aware of? Or I have to split the sentence first, reverse the order of the 2nd part, and then model the two parts using two rnn networks respectively (which is what I am doing now). 

Thanks!!

**Comment by tatatodd** [comment]
Thanks for the question @bluemonk482

We primarily use github issues to track bugs, installation issues, and feature requests.   This is a question better suited for StackOverflow.  Please ask it there and tag it with the `tensorflow` tag.  Thanks!
